pkg/kv/kvserver/store_snapshot.go

```diff
@@ -66,16 +66,6 @@ const (
 	DefaultSnapshotApplyLimit = 1
 )
 
-// snapshotPrioritizationEnabled will allow the sender and receiver of snapshots
-// to prioritize the snapshots. If disabled, the behavior will be FIFO on both
-// send and receive sides.
-var snapshotPrioritizationEnabled = settings.RegisterBoolSetting(
```

Comment:
I don't have context on this part of the change. Basically a testing-only setting unlikely to have been used by anyone?

Reply:
This was a backup we had put in in case the snapshot prioritization in 22.2 had any issues. It has been running for a while now and seems good.